feature-libs/organization/order-approval/components/details/order-approval-details.module.ts

```diff
@@ -62,7 +62,7 @@ import { OrderDetailPermissionResultsComponent } from './order-detail-permission
           component: OrderDetailPermissionResultsComponent,
         },
         OrderApprovalDetailShippingComponent: {
-          component: OrderDetailShippingComponent,
```

Comment:
Should be documented in the ts manual breaking change doc for hicher level changes like this.

Reply:
Added